extensions/sql-database-projects/package.json

```diff
@@ -60,11 +153,15 @@
   },
   "dependencies": {
     "vscode-nls": "^3.2.1",
-    "xml2js": "^0.4.23"
+    "xmldom": "^0.3.0",
+    "vscode-languageclient": "^5.3.0-next.1"
   },
   "devDependencies": {
-    "@types/xml2js": "^0.4.5",
+    "@types/xmldom": "^0.1.29",
+    "should": "^13.2.1",
     "tslint": "^5.8.0",
-    "typescript": "^2.6.1"
+    "typescript": "^2.6.1",
+    "typemoq": "^2.1.0",
+    "vscodetestcover": "github:corivera/vscodetestcover#1.0.6"
```

Comment:
possible that the PR build error you is because of not adding "mocha-multi-reporters": "^1.1.7", in dev dependencies?

Reply:
Yup, you need to add a dependency for that + "mocha-junit-reporter": "^1.17.0"

https://github.com/microsoft/azuredatastudio/blob/master/extensions/agent/package.json#L88